src/Accelerators/NNPA/Conversion/ONNXToZHigh/RewriteONNXForZHigh.cpp

```diff
@@ -180,38 +180,12 @@ bool isUniBroadcatableFirstToSecond(Value A, Value B) {
 
 /// Check a value is defined by ONNXConstantOp or not.
 bool isDefinedByONNXConstantOp(Value v) {
+  // @Tung: why does it matters? Isn't a constant a constant?
```

Comment:
this is needed because `v.getDefiningOp() `returns null if v is a BlockArgument and `isa<ONNXConstantOp>(x)` crashes if x is null

but it's not needed if you use `isa_and_present`

Reply:
Yes, a block argument does not have a defining op, so it would be null.

Reply:
Changing to isa_and_present as it would appear to cover more cases (all where its null)

Reply:
please remove the comment